### DISCUSSION TITLE
Creating new theme keys

### DISCUSSION THREAD

**Discussion by nicoled-performio** [chakra-ui/chakra-ui#6230] [Q&A]
Hi, is there any way to create new theme keys so we can add more tokens to our base theme. For example opacity currently does not have a theme key. Is there a way to extend theme to include an opacity theme key?

**Comment by anubra266** [comment]
@Nic-Performio You can't add new keys to the theme. But this could be useful for what you want to do. :https://chakra-ui.com/docs/styled-system/text-and-layer-styles#layer-styles
